### DISCUSSION TITLE
Admob just showing test ads

### DISCUSSION THREAD

**Discussion by bertangokalp** [Shin-NiL/Godot-Android-Admob-Plugin#211] [Q&A]
For 2 months, only test ads appear in my application, I searched all documents, it is true in isReal, but I couldn't find any results. Can anyone help?
Godot Version: 3.5.1

**Comment by Omstero** [comment]
Was there a solution to your problem? Kinda have the same right now.

**Reply by Omstero** [reply]
I actually fixed it. First i set is_real to true evrywhere in the code. But that alone was not the Problem. The thing that actually fixed it was exporting the game with the Adbmob Plugin disabled in the export menu. Then I enabled it again, and in that version it worked. No more test labels but real Ads. I guess it just needed to be forced to compile the whole plugin again.

**Comment by Omstero** [comment]
I found out that it seems to only happen in the .aab export. The local .apk works just fine and shows real ads without the test ads label. But i dont know why this could be the case. My .aab file is definitely not a debug version.
